### ISSUE TITLE
About  subsample and produce triplet.txt

### ISSUE THREAD

**Issue by longzeyilang** [akarshzingade/image-similarity-deep-ranking#16]
Hi, According to  the original，I have two question below:
1) the original use muti-scale model, you demo can not use, why?
2) the original use pairwise relevance scores to choose the better positive and negative image sample, in this demo, do not use the mentioned rules. why?
Thanks a lot!

**Comment by akarshzingade** [comment]
Hey Longzeyilang! 

1) Do you mean the parallel smaller network with VGG? If yes, I do have implemented it :)
2) The pairwise relevance score is hand crafted. Unfortunately, they haven't open-sourced that information so couldn't use it.

**Comment by longzeyilang** [comment]
Thank you!  I have another problem, the returned value _batch_y_ of  _train_generator_  function not  use in the train.  if batch_y is all the same, it does not matter!

**Comment by akarshzingade** [comment]
Yes, that is correct! I was supposed to change this. But, somehow forgot about it! Thank you for pointing that out :)
